packages/playwright/src/generators/init/generator.ts

```diff
@@ -135,5 +135,5 @@ export default async function initGenerator(
 
     await formatFiles(tree);
 
-    return updateDependencies(tree);
+    return options.accessibility && updateDependencies(tree);
```

Comment:
I don't really like this pattern, I'd prefer to move this conditional into `updateDependencies` so it's more future proof

Reply:
updated